prometheus/go_collector.go

```diff
@@ -8,6 +8,11 @@ type goCollector struct {
 	goroutines Gauge
 }
 
+type gcCollector struct {
```

Comment:
Ah, the intention was to make it part of the `goCollector` instead of having a separate `gcCollector` (not just move it to the same file).